haddock-api/haddock-api.cabal

```diff
@@ -34,6 +34,11 @@ data-files:
   html/Ocean.std-theme/ocean.css
   html/Ocean.std-theme/plus.gif
   html/Ocean.std-theme/synopsis.png
+  html/NewOcean.std-theme/hslogo-16.png
+  html/NewOcean.std-theme/minus.gif
+  html/NewOcean.std-theme/new-ocean.css
+  html/NewOcean.std-theme/plus.gif
+  html/NewOcean.std-theme/synopsis.png
```

Comment:
Sorry if I am annoying (it's much easier to comment than doing it myself), but for the synopsis and "minus" I moved to use unicode instead of an actual imagine.

Reply:
Not at all - please continue to drop in and comment on this sort of thing!

I did notice that the `synopsis` button is still the one of old. (And looks funky, since the old button was designed to hang off the right side of the screen). I'll investigate later.